spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/InterpreterSuite.scala

```diff
@@ -86,7 +86,7 @@ class InterpreterSuite extends AnyFunSuite with SparkTestBase with BeforeAndAfte
     spark.disableControlMeasuresTracking()
 
 
-    eventually(timeout(scaled(10.seconds)), interval(scaled(500.millis))) {
+    eventually(timeout(scaled(15.seconds)), interval(scaled(500.millis))) {
```

Comment:
Just curious: why the change?

Reply:
It can be quite slow (on battery, e.g.). Mainly on Jenkins, for which I increased the scale factor, but to make sure there is enough time.